src/nvim/eval.c

```diff
@@ -4523,7 +4523,7 @@ static int eval_index(char_u **arg, typval_T *rettv, int evaluate, int verbose)
      * dict.name
      */
     key = *arg + 1;
-    for (len = 0; ASCII_ISALNUM(key[len]) || key[len] == '_'; ++len) {
+    for (len = 0; ASCII_ISALNUM(key[len]) || key[len] == '_'; len++) {
```

Comment:
Even if uncrustify could fix this, maybe it shouldn't?
Pre/post increment depends on the context of the code.

Reply:
Uncrustify can't fix this at the moment. The uncrustify devs wouldn't accept an option that blatantly changes all pre-increments to post-increments, it'd have to be only the ones in that's used in a statement and not in an expression.